### PR TITLE
CAT-639 More risk profiler vulnerabilities

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -104,29 +22,23 @@ dependencies {
...
-    implementation 'net.javacrumbs.shedlock:shedlock-provider-jdbc-template:4.10.1'
-
-    implementation 'net.logstash.logback:logstash-logback-encoder:6.3'
-    implementation 'com.microsoft.azure:applicationinsights-spring-boot-starter:2.6.0'
-    implementation 'com.microsoft.azure:applicationinsights-logging-logback:2.6.0'
-    agentDeps 'com.microsoft.azure:applicationinsights-agent:2.6.0'
+    implementation("org.springframework.security.oauth:spring-security-oauth2:2.5.0.RELEASE")
+    implementation("org.springframework.security:spring-security-jwt:1.1.1.RELEASE")
+
+    compile group: 'org.hibernate.validator', name: 'hibernate-validator', version: '6.1.5.Final'
```
Comment (petergphillips) [inline build.gradle:33]: `compile` deprecated - replaced with `implementation`.
Should really be
```
    implementation 'org.hibernate.validator:hibernate-validator:6.1.5.Final'
```
do you really need it at all though - isn't it included with `gradle-spring-boot`?

File: build.gradle [hunk 2]
```diff
@@ -139,31 +51,19 @@ dependencies {
...
     testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
     testCompileOnly 'org.projectlombok:lombok:1.18.12'
-    testImplementation("org.springframework.boot:spring-boot-starter-test")
-    testImplementation("org.assertj:assertj-core:3.16.1")
     testImplementation 'io.github.http-builder-ng:http-builder-ng-apache:1.0.4'
-
-    testImplementation 'org.apache.camel:camel-test-spring:2.25.1'
+    testImplementation 'org.apache.camel:camel-test-spring:3.3.0'
 
     compile configurations.agentDeps
```
Comment (petergphillips) [inline build.gradle:68]: don't need this line

File: build.gradle (hunk 1 above)
Comment (steverendell) [reply build.gradle]: yup, its ok without it now

File: build.gradle (hunk 2 above)
Comment (steverendell) [reply build.gradle]: 👍

**Pull request merged**